vcpkg-overlays/linux-syslibs/ubuntu20/zlib/portfile.cmake

```diff
@@ -0,0 +1 @@
+set(VCPKG_POLICY_EMPTY_PACKAGE enabled)
```

Comment:
Can ubuntu 20 use the default openssl port? The intent of this port directory structure is for distros with sufficient package versions to use "default" - otherwise we end up making a new port for each and every distro that we build on.